android/KMAPro/kMAPro/src/main/java/com/keyman/android/SystemKeyboard.java

```diff
@@ -244,6 +245,9 @@ public void onKeyboardLoaded(KeyboardType keyboardType) {
       if (exText != null)
         exText = null;
     }
+    // Initialize the longpress delay
+    int longpressDelay = KMManager.getLongpressDelay();
+    KMManager.applyLongpressDelay(longpressDelay);
```

Comment:
I am not very comfortable with this. KMManager should be able to manage the value internally, and we shouldn't be passing the value back and forth. If we need to ask KMManager to re-apply the longpressDelay that it already knows, it should be just something like:

```suggestion
    KMManager.applyLongpressDelay();
```